ipatests/test_integration/test_cert.py

```diff
@@ -96,6 +99,18 @@ def test_ipa_getcert_san_aci(self):
         certfile = '/etc/pki/tls/certs/test2.pem'
 
         tasks.kinit_admin(self.master)
+
```

Comment:
Why we are just logging the failure from reverse zone addition if any? It should mark the test fail and move to next test.
If this section of test just needs to be executed on a particular platform then we can use tasks.get_platform(self.master) to run this on that particular platform.